networkmanager/networkmanager_test.go

```diff
@@ -415,6 +449,93 @@ func TestBandwithPlugin(t *testing.T) {
 	}
 }
 
+func TestUpdateNetwork(t *testing.T) {
+	networkParameters := []aostypes.NetworkParameters{
+		{
+			VlanID:    10,
+			NetworkID: "network0",
+			IP:        "172.17.0.1",
+			Subnet:    "172.17.0.0/16",
+		},
+		{
+			VlanID:    11,
+			NetworkID: "network1",
+			IP:        "172.18.0.1",
+			Subnet:    "172.18.0.0/16",
+		},
+	}
+
+	cniInterface := &testCNIInterface{}
+
+	networkmanager.CNIPlugins = cniInterface
+	storage := testStorage{
+		chains:            make(map[string]trafficData),
+		netData:           make(map[string]networkmanager.NetworkInfo),
+		chanRemoveNetwork: make(chan struct{}, 2),
```

Comment:
Why size is 2? Make it as const?

Reply:
I want to create two networks and since two operations are performed within the UpdateNetworks function, I want to handle them through channels. To avoid blocking, I buffer them to a size of 2.
added const variable